### PR TITLE
Adding classes for report submenu to make it look more like the rest.…

### LINKED ISSUE #75: Reports Menu Turns White in Admin
When clicking in the reports menu and off of it again it obtains this white color it is very hard to see.  I've attached a screen shot of it.

report_menu_color

### PR DESCRIPTION
**Pull request by cam156**
… fixes #75

Original:

screen shot 2016-04-05 at 12 20 25 pm

New with CSS
screen shot 2016-04-05 at 12 19 47 pm

### CODE REVIEW

File: app/assets/stylesheets/_base.css.scss [hunk 1]
```diff
@@ -428,7 +428,7 @@ select.disabled {
   margin: 0.5em auto;
 }
 
-.nav-pills > li.active > a, .nav-pills > li > a:hover, .nav-pills > li.active > a:hover {
+.nav-pills > li.active > a, .nav-pills > li > a:hover, .nav-pills > li.active > a:hover, .nav > li > a:focus {
```
Comment (mtribone) [inline app/assets/stylesheets/_base.css.scss]: You override .nav > li > a:focus on line 446. Is it needed here?

File: app/assets/stylesheets/_base.css.scss [hunk 2]
```diff
@@ -436,3 +436,31 @@ select.disabled {
...
+}
+
+.nav > li > a:focus {
+  background-color: $medium-blue-bright;
+  color: white;
+  border-radius: $border-radius-large;
+}
+
+.report-buttons li a {
+  color: white;
```
Comment (mtribone) [inline app/assets/stylesheets/_base.css.scss]: Maybe a variable here, too?

File: app/assets/stylesheets/_base.css.scss (hunk 1 above)
Comment (cam156) [inline app/assets/stylesheets/_base.css.scss]: @mtribone good call this change is not needed.

**Comment by cam156** [comment]
@jonibar I think this is ready to merge once it is green...

**Comment by jonibar** [comment]
You’re welcome to merge it

**Pull request merged**